code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -769,16 +769,16 @@ var/global/list/obj/item/device/pda/PDAs = list()
 		empulse(P.loc, 3, 6, 1)
 		message += "Your [P] emits a wave of electromagnetic energy!"
 	if(i>=25 && i<=40) //Smoke
-		var/datum/effect_system/smoke_spread/chem/S = new /datum/effect_system/smoke_spread/chem
+		var/datum/effect_system/smoke_spread/S = new
```

Comment:
Just `new` ?

Reply:
Shamefully copied from the DM help section:

> if Type is not specified and new() is being used in an assignment, the variable type of the left-hand-side will be used as the default type.

So, yea, just the `new`.